vimmasite/vimma/celeryconfig.py

```diff
@@ -5,6 +5,10 @@
 _every_5min = crontab(minute='*/5')
 _every_1h = crontab(minute=0)
 
+BROKER_BACKEND = "redis"
```

Comment:
Why use redis, instead of the default rabbitmq? Was there some problem getting it running?